### PR TITLE
PARIS - Devsquad 107 Aurelie & Jules

### PR DESCRIPTION
**Pull request by julesbadin**
last iteration to be done

### CODE REVIEW

**Comment by julesbadin** [comment]
@AurelieBeno

File: starter-code/pizza.js [hunk 1]
```diff
@@ -1 +1,72 @@
...
+var totalPriceArr = ["10"];
+var totalPrice = $(".strong");
+var itemPriceArr = [];
+var itemPrice = $(".price li");
+
+// var panelControl.forEach(function (oneTopping) {
+//   if (oneTopping.hasClass("active")) {
+//     console.log("coucou toppingitem");
+//   } else {
+//     console.log("sa mere");
```
Comment (ta-web-paris) [inline starter-code/pizza.js:69]: I like your console.log!

File: starter-code/pizza.js [hunk 2]
```diff
@@ -1 +1,72 @@
...
+});
+
+
+//----Total Price -----
+
+var totalPriceArr = ["10"];
+var totalPrice = $(".strong");
+var itemPriceArr = [];
+var itemPrice = $(".price li");
+
```
Comment (ta-web-paris) [inline starter-code/pizza.js:64]: Here is how you can update your prices with a function that you add in each click function and you update your price each time you load the page: 
```
function updatePriceTotal() {
  var total = 10;

  $(".price li:visible").each(function() {
    var desc = $(this).html();
    var additional = parseInt(desc.slice(1, 2), 10);
    total += additional;
  });

  $(".price strong").html("$" + total);
}

$(window).on("load", updatePriceTotal);

```

**Comment by ta-web-paris** [comment]
Very good. Good understanding of jQuery! Continue your good work!

**Pull request closed without merging**